### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by drewdrewthis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,110 @@
...
+  `yarn add athenaeum` or `npm install athenaeum`
+
+
+## Using an RCL component:
+
+RCL components are React components, which are exported as a collection of modules you can import/require like you would any other JS dependency:
+
+e.g.
+
+```
```
Comment (jmcolella) [inline README.md]: use ````js` to get syntax highlighting in code block

Comment (drewdrewthis) [inline README.md]: @jmcolella haha, I had it, but your linter being annoying about it.

Comment (jmcolella) [inline README.md]: @drewdrewthis I can review the errors and probably just add them as exclusions to our plugin

Comment (drewdrewthis) [inline README.md]: `51:52  error  Missing file extension for "athenaeum"  import/extensions`

File: README.md [hunk 2]
```diff
@@ -1,39 +1,145 @@
...
+      <Button>Submit!</Button>
+    </Layout>
+  </div>
+  )
+}
+
+export default MyReactComponent;
+```
+
+To use the RCL styles, you can either import them directly into your project:
```
Comment (patrickkim) [inline README.md]: Care to expound further?

File: README.md [hunk 3]
```diff
@@ -1,39 +1,145 @@
...
 ```
+
+Our RCL is structured around the concepts put forth by [Atomic Design](http://bradfrost.com/blog/post/atomic-web-design/). The smallest components are **ATOMS**, which are used to make **MOLECULES**, which are used to make **ORGANISMS**, which are lastly used to make **TEMPLATES**.
+
+
+## Contributing
+
+### Coding styles and standards
+
+Most of our styles are enforced by linters ([stylelint](https://github.com/stylelint/stylelint), [eslint](http://eslint.org/), [eslint-plugin-markdown](https://github.com/jmcolella/eslint-plugin-markdown)), which must follow for the builds to pass.
```
Comment (patrickkim) [inline README.md]: Nice plug @jmcolella

File: README.md [hunk 4]
```diff
@@ -1,39 +1,145 @@
...
+
+We use [Buildkite](https://buildkite.com/policygenius) as a CI server. Every commit triggers a build with Buildkite that will run a series of checks and tests before giving a green light to merge to master.
+
+There is 1 pipelines associated with this project:
+
+[athenaeum pipeline](https://buildkite.com/policygenius/athenaeum) - builds the project, runs test. When a branch is then merged to master, a new build is triggered. After a build on master passes all of its, it will need to be manually unblocked before being deployed to production and published to npm. This manual unblock is a prompt requiring you to bump the package version, for which we use [Semantic Versioning](http://semver.org/).
+
+### Code reviews
+
+Your branch code should always be reviewed before it. **Do not work on the `master` branch.**
```
Comment (patrickkim) [inline README.md]: before it ...

File: README.md (hunk 2 above)
Comment (drewdrewthis) [inline README.md]: @patrickkim Haha, do you? On what do you need me to expound?

Comment (drewdrewthis) [inline README.md]: Do you mean that I should explicitly say "CSS" instead of "styles'?

File: README.md (hunk 4 above)
Comment (drewdrewthis) [inline README.md]: 👍

**Pull request merged**